apps/test/unit/code-studio/progressReduxTest.js

```diff
@@ -1,16 +1,263 @@
-import { assert } from 'chai';
+import { assert } from '../../util/configuredChai';
```

Comment:
do you actually need to import configuredChai? or does the chai configuration apply globally?